docs/tutorials/using_pyrokinetics_on_hpc_systems.rst

```diff
@@ -0,0 +1,395 @@
+           help="Gyrokinetics input file used as basis for all runs",
+       )
+       parser.add_argument(
+           "eq_files",
+           type=Path,
+           nargs="+",
+           help="GEQDSK equilibrium files to simulate",
+       )
+       parser.add_argument(
+           "--psi",
+           type=float,
+           default=0.5,
+           help="Normalised psi at which to generate flux surfaces",
+       )
+       return parser.parse_args()
+
+
+   def main() -> None:
+       # Get command line arguments
+       args = parse_args()
+
+       # For each equilibrium file, get the path of a corresponding new GS2 input file
+       # in its own directory
+       eq_files = [path.resolve() for path in args.eq_files]
+       new_dirs = [path.parent / f"{path.name}.d" for path in eq_files]
+       gk_files = [d / f"{path.stem}.in" for d, path in zip(new_dirs, eq_files)]
+
+       # Set up job queue
+       jobs = Jobs()
+
+       # Generate new input file for each equilibrium file, add to job queue
+       gs2 = args.gs2_exe.resolve()
+       template = args.gk_file.resolve()
+       for gk_file, eq_file in zip(gk_files, eq_files):
+           pyro = Pyro(gk_file=template, eq_file=eq_file)
+           pyro.load_local_geometry(psi_n=args.psi)
+           print("Generating input file:", gk_file)
+           pyro.write_gk_file(gk_file, gk_code="GS2")
+           jobs.add(
+               name=str(gk_file.stem),         # Name each job
+               exec="srun",                    # Run using srun...
+               args=[str(gs2), str(gk_file)],  # ...with the GS2 exe and each input file
+               stdout=str(gk_file.parent / f"{gk_file.stem}.log"),  # Log file name
+               stderr=str(gk_file.parent / f"{gk_file.stem}.err"),  # Error file name
+               numCores=8,                     # How many tasks per run
+           )
+
+       # Submit jobs and print stats
+       manager = LocalManager()
+       print("Available resources:", manager.resources())
+       job_ids = manager.submit(jobs)
+       print("Submitted jobs:", job_ids)
+       job_status = manager.status(job_ids)
+       print("Job status:", job_status)
+
+       # Wait for jobs to complete and print final status
+       manager.wait4(job_ids)
+       job_info = manager.info(job_ids)
+       print("Job detailed information:", job_info)
+       manager.finish() # NB: This is needed!
+
+
+   if __name__ == "__main__":
+       main()
+
+This can then be run with the batch script ``~/scratch/pyro_job/pyro_job.sh``:
+
+.. code-block:: bash
+
+   #!/bin/bash
+
+   # Slurm settings
+   # --------------
+
+   #SBATCH --job-name=pyro_job      # Job name
+   #SBATCH --mail-user=me@email.com # Where to send mail reporting on the job
+   #SBATCH --mail-type=END,FAIL     # Mail events (NONE, BEGIN, END, FAIL, ALL)
+   #SBATCH --nodes=1                # Number of compute nodes to run on
+   #SBATCH --ntasks-per-node=24     # Number of MPI processes to spawn per node
```

Comment:
Difference wrt simple case -- might be worth keeping this the same as before to avoid "noise"

Reply:
Ah I see from the python that you're asking each job to use 8 cores so I guess  the intention here is to run all three cases at the same time -- might be worth some text up front to indicate (e.g. before the python script) to say what approach to running you're targeting here?